metricbeat/module/apache/_meta/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG APACHE_VERSION=${APACHE_VERSION}
 FROM httpd:$APACHE_VERSION
-RUN sed -i "/jessie-updates/d" /etc/apt/sources.list
-RUN apt-get update && apt-get install -y curl
+RUN apt-key list  | grep "expired: " | sed -ne 's|pub .*/\([^ ]*\) .*|\1|gp' | xargs -n1 apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys
```

Comment:
There is a special test for an older Apache version and I'd like to keep it. So, to run Debian Jessie we have to update the keys as an additional step now.